tests/client.py

```diff
@@ -773,6 +781,111 @@ def create(self, stream, company_ids=[], subscriptions=[], times=1):
+             get_url = f"{BASE_URL}/properties/v1/contacts/properties/named/" + property
+             response = self.get_custom_property(get_url)
+             if ( response.status_code == 404 ):
+                 continue
+             url = f"{BASE_URL}/properties/v1/contacts/properties/named/" + property
+             response = self.delete(url)
+             LOGGER.info("response is %s", response)
+
+    def create_custom_contact_properties(self):
+        """Create custom contact properties of all the types"""
+        self.delete_custom_contacts_property()
+
+        url = f"{BASE_URL}/properties/v1/contacts/properties"
+        data1 = {
+            "name": "custom_string",
+            "label": "A New String Custom Property",
+            "description": "A new string property for you",
+            "groupName": "contactinformation",
+            "type": "string",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 6,
+            "options": [
+            ]
+        }
+
+        data2 = {
+            "name": "custom_number",
+            "label": "A New Number Custom Property",
+            "description": "A new number property for you",
+            "groupName": "contactinformation",
+            "type": "number",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 7,
+            "options": [
+            ]
+        }
+
+        data3 = {
+            "name": "custom_date",
+            "label": "A New Date Custom Property",
+            "description": "A new date property for you",
+            "groupName": "contactinformation",
+            "type": "date",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 9,
+            "options": [
+            ]
+        }
+
+        data4 = {
+            "name": "custom_datetime",
+            "label": "A New Datetime Custom Property",
+            "description": "A new datetime property for you",
+            "groupName": "contactinformation",
+            "type": "datetime",
+            "fieldType": "text",
+            "formField": True,
+            "displayOrder": 10,
+            "options": [
+            ]
+        }
+        data5 = {
+            "name": "multi_pick",
+            "label": "multi pick",
+            "description": "multi select picklist test",
+            "groupName": "contactinformation",
+            "type": "enumeration",
+            "fieldType": "checkbox",
+            "hidden": False,
+            "options": [
+              {
+                "label": "Option A",
+                "value": "option_a"
+              },
+              {
+                "label": "Option B",
+                "value": "option_b"
+              },
+              {
+                "label": "Option C",
+      "          value": "option_c"
+              }
+            ],
+            "formField": True
+        }
+
+        # generate a contacts record
```

Comment:
Dev has recommended using raise_for_status() after requests.  As written we will just log bad responses and continue the test.  Is this the desired behavior?

Reply:
post method is doing the raise_for_status check. create_contacts method is doing similar to what is beoing done here. I took that as the template.